c2_store/Android.bp

```diff
@@ -21,12 +21,18 @@ cc_library_static {
 }
 
 cc_binary {
-
-    name: "hardware.intel.media.c2@1.0-service",
-    defaults: ["libcodec2-hidl-defaults", "mfx_c2_exe_defaults"],
+    name: "android.hardware.media.c2-service.intel",
```

Comment:
hi Jeeva, could you please share why we rename the service name here? Is there any naming rule defined in Android 15?

Reply:
aidl naming convention removed the version information. Thats the reason behind the name change to android.hardware.media.c2-service.intel